controllers/etcd_controller.go

```diff
@@ -956,40 +972,56 @@ func (r *EtcdReconciler) updateEtcdErrorStatus(etcd *druidv1alpha1.Etcd, lastErr
 	lastErrStr := fmt.Sprintf("%v", lastError)
 	etcd.Status.LastError = &lastErrStr
 	etcd.Status.ObservedGeneration = &etcd.Generation
+	etcd.Status.Ready = false
```

Comment:
What is the purpose of the `ready` field? If it is to show whether the etcd statefulset is fine, then I suggest to change it to

```suggestion
	etcd.Status.Ready = health.CheckStatefulSet(ss) == nil
```